client/blocks/eligibility-warnings/index.tsx

```diff
@@ -330,20 +332,27 @@ EligibilityWarnings.defaultProps = {
  * 'NO_BUSINESS_PLAN' hold on atomic transfer; however, if we're about to buy a
  * marketplace addon which provides the ATOMIC feature, then we can ignore this
  * hold.
+ *
+ * The Onboarding flow takes care of upgrading the Business plan so we can
+ * can ignore the 'NO_BUSINESS_PLAN' hold.
  */
 const processMarketplaceExceptions = (
 	state: Record< string, unknown >,
 	eligibilityData: EligibilityData,
-	isEligible: boolean
+	isEligible: boolean,
+	isOnboarding?: boolean
 ) => {
 	// If no eligibilityHolds are defined, skip.
 	if ( typeof eligibilityData.eligibilityHolds === 'undefined' ) {
 		return { eligibilityData, isEligible };
 	}
 
-	// If missing INSTALL_PURCHASED_PLUGINS feature, skip.
+	// If not explicit exception and missing INSTALL_PURCHASED_PLUGINS feature, skip.
```

Comment:
Please update this comment :)

Reply:
Done 😄 https://github.com/Automattic/wp-calypso/pull/81412/commits/04c707f70a4cf43334ae85102d8d7cc24a6c8f0b